src/services/userStatusData.service.js

```diff
@@ -0,0 +1,67 @@
+import { toast } from 'react-toastify';
+import UserWeekStats from 'src/classes/UserWeekStats';
+import { UserService } from './user.service';
+import { weekService } from './week.service';
+import { userWeekStatsService } from './userWeekStats.service';
+import { defaultStatsValues } from 'src/constants/studentsTable';
+
+
+const getAllByProgram = async (program = '1') => {
```

Comment:
The function should not get as a default value a "hard-coded" data that is defined in the DB. 
It should know how we use it but not the data itself.